File: AppMantemiento_mecanico/AppDiagramaClase/conexion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MySql.Data.MySqlClient;
using System.Data;

namespace AppDiagramaClase
{
    class conexion
    {
        public static MySqlConnection conectar()
        {
            string server = "127.0.0.1";
            string database = "Mantenimiento";
            string user = "root";
            string pass = "";

            MySqlConnection cn = new MySqlConnection("Server="+server+";Database="+database+";User="+user+";Pwd="+pass);
            return cn;
        }

    }
}

File: AppConsultas_medica/AppConsultas_medica/Modelos/Conexion.vb
Imports MySql.Data.MySqlClient
Imports System.Data
Public Class Conexion

    Public Shared Function conectar() As MySqlConnection

        Dim servidor As String = "127.0.0.1"
        Dim basedatos As String = "consulta_medicas"
        Dim usuario As String = "root"
        Dim contrasenia As String = ""

        Dim cn As New MySqlConnection("Server=" + servidor + ";Database=" + basedatos + ";User=" + usuario + ";Pwd=" + contrasenia)
        Return cn

    End Function
End Class